src/server/game/Handlers/ChatHandler.cpp

```diff
@@ -236,14 +243,6 @@ void WorldSession::HandleMessagechatOpcode(WorldPacket & recvData)
             sender->UpdateSpeakTime(specialMessageLimit);
     }
 
-    // pussywizard: optimization
```

Comment:
how was this an optimization???

Reply:
That wasn't an optimization xD

Reply:
it's just that nobody use this spell but you I guess xD

Reply:
exactly, if you don't use that spell, it's an "optimization" removing that condition, but only if you are sure that you do not use never the related spell.

Reply:
so we reintroduce it or not? xD

Reply:
I already use this spell, When I see a advertiser in my server xD